readme.md

```diff
@@ -56,20 +68,17 @@
     - C++ /CLI support for v143 build Tools (Latest)  (for OMOD and FOMOD installers)
     - Windows 11 SDK (get latest)
     - C++ Build Tools core features
-  - Optional:
-    - Git for Windows (if you don't have git already installed)
-- Start a shell
-  - If you have git in your path, a regular cmd shell works.
-  - If not, start the _x64 Native Tools Command Prompt for VS 2019_, it'll use Git for Windows installed above.
-- Do:
-	```
-	mkdir C:\dev
-	cd C:\dev
-	git clone https://github.com/ModOrganizer2/mob
-	cd mob
-	bootstrap
-	mob -d C:\dev\modorganizer build
-	```
+
+## Setting up MOB
+- Start the _x64 Native Tools Command Prompt for VS 2022, as Administrator, we need this for cmake & git (not needed if you have either available in your PATH)
```

Comment:
We *definitely* shouldn't need admin permissions, and using them is more likely to introduce problems (if *you've* needed admin permissions, it's likely because you've previously run something with admin permissions you weren't supposed to, and that's meant a file or directory was created with admin ownership). If you're not running software that was originally written for Windows XP or earlier, running things as admin absolutely shouldn't be in your toolbox as a troubleshooting step. If you're hitting problems, the software's got a bug (or you're using it incorrectly), and giving potentially buggy software greater powers to ruin your system is like giving a toddler an uzi because they're shooting a nerf gun at things they're not supposed to - it's dangerous and there's no logical reason to expect it'd help.

I also don't particularly like the swap between a regular shell being the default and the MSVC-activated one being the default. I'm confident it's a minority of people who use the Microsoft repacks of Git for Windows and CMake that are provided by the Visual Studio installer instead of the official distributions, and it's only those people who benefit from the x64 Native Tools Command Prompt (or any of its siblings) as the regular Git and CMake installers put them on the PATH, and if you remove them, using the MSVC shell won't put them back.

Reply:
I second this, there are no reasons to either use an administrator prompt, and using the VS command prompt just to get the shipped CMake / Git is a bit overkill.

Reply:
Visual Studio doesn't install cmake to the windows PATH when selected in the VS installer, which causes mob to outright fail. 

I've assumed Git is the same, but I could be wrong. I agree we should be advising the use of the modern terminal, but that means re-writing a large part of the readme to switch to advising using package managers like `winget` or `choco` instead of using the VS installer to install the needed/suggested tools.

Reply:
Updated with some clarifications

Reply:
There is no suggestion to rely on VS for Git or CMake (except for the optional stuff) in the README. We should add git and CMake as dependencies before Qt or anything.

We do not have to advise installing tools with `winget`, `choco`, custom installer, or whatever - Just listing the required tools is enough (that also applies for my `aqt` comment). If someone does not manage to install `git` or `cmake` when told to, I think they're going to face some issues trying to work on MO2.

Reply:
> Visual Studio doesn't install cmake to the windows PATH when selected in the VS installer, which causes mob to outright fail when the VS x64 native tools cmd isn't used, unless CMake is installed outside of the VS installer.

We know this, but we're saying it's only a minority of people who'll follow this guide who'll have installed these tools from VS, so a minority who'll potentially hit this problem. It therefore makes sense to treat having the tools on the path as the default case, and needing the VS shell as the weird one.

*Start the _x64 Native Tools Command Prompt for VS 2022, we need this for cmake & git (you can use a regular terminal if you've installed both outside of the VS installer, or have both available in your PATH)*

is worse than

*Open a shell. If you've installed Git or CMake through the VS installer, they won't be on your PATH, so you must use *x64 Native Tools Command Prompt for VS 2022*.*

> We do not have to advise installing tools with winget, choco, custom installer, or whatever - Just listing the required tools is enough (that also applies for my aqt comment). If someone does not manage to install git or cmake when told to, I think they're going to face some issues trying to work on MO2.

When I'm writing this kind of guide, I do tend to list the names of chocolatey packages for any dependency I mention. While I expect people following the guide to be able to install these tools, it's much less hassle to copy and paste a package name than to search for a package and check it's the same tool as the guide wants.

Reply:
I don't really see a difference between either statements, regardless I've pushed another update to appease.

Reply:
> > We do not have to advise installing tools with winget, choco, custom installer, or whatever - Just listing the required tools is enough (that also applies for my aqt comment). If someone does not manage to install git or cmake when told to, I think they're going to face some issues trying to work on MO2.
> 
> When I'm writing this kind of guide, I do tend to list the names of chocolatey packages for any dependency I mention. While I expect people following the guide to be able to install these tools, it's much less hassle to copy and paste a package name than to search for a package and check it's the same tool as the guide wants.

I don't like adding instructions for a specific package manager because it tends to make it look like you need that particular package manager, which is clearly not the case. And adding instructions for every relevant package managers out there (+ manual installation) is not something we want.

Reply:
Chocolatey's the most widely used, so is a shoo-in. Scoop is less popular and generally has a less complete library, so can be skipped. WinGet has Microsoft's backing, so might be worth including. Everything else is so niche that no one will care if it's excluded.